### ISSUE TITLE
Avocado crashes during remote execution of builtin tests

### ISSUE THREAD

**Issue by fkromer** [avocado-framework/avocado#2326]
I try to execute tests on a remote machine. On the remote machine and on the "triggering" machine I installed `avocado` via `pip`. In addition to that I installed [avocado-framework-plugin-runner-remote](https://pypi.python.org/pypi/avocado-framework-plugin-runner-remote) on the "triggering" machine with `sudo pip install avocado-framework-plugin-runner-remote`. But when I execute the simple failing test it is not executed and avocado crashes on the "triggering" machine:

```
$ sudo avocado run --remote-hostname 10.0.2.93 --remote-username rc --remote-password <password> /usr/share/avocado/tests/failtest.py:FailTest.test
[sudo] password for kromer: 
JOB ID     : 9139a1ea991fade4a80b7a5a732ab5b522455cca
JOB LOG    : /home/kromer/avocado/job-results/job-2017-12-05T16.29-9139a1e/job.log
LOGIN      : rc@10.0.2.93:22 (TIMEOUT: 60 seconds)

Avocado crashed: AttributeError: 'Namespace' object has no attribute 'mux_yaml'
Traceback (most recent call last):

  File "/usr/local/lib/python2.7/dist-packages/avocado/core/job.py", line 474, in run_tests
    execution_order)

  File "/usr/local/lib/python2.7/dist-packages/avocado_runner_remote/__init__.py", line 501, in run_suite
    results = self.run_test(self.job.references, timeout)

  File "/usr/local/lib/python2.7/dist-packages/avocado_runner_remote/__init__.py", line 414, in run_test
    mux_files = getattr(self.job.args, 'mux_yaml') or []

AttributeError: 'Namespace' object has no attribute 'mux_yaml'

Please include the traceback info and command line used on your bug report
Report bugs visiting https://github.com/avocado-framework/avocado/issues/new
```

When the test is executed directly on the remote machine the test fails just fine:

```
avocado run /usr/share/avocado/tests/failtest.py:FailTest.test
JOB ID     : 075485d5e70fc43d17d7844bd7d5033e8e1c488f
JOB LOG    : /home/rc/avocado/job-results/job-2017-12-05T15.29-075485d/job.log
 (1/1) /usr/share/avocado/tests/failtest.py:FailTest.test: FAIL (0.82 s)
RESULTS    : PASS 0 | ERROR 0 | FAIL 1 | SKIP 0 | WARN 0 | INTERRUPT 0 | CANCEL 0
JOB TIME   : 5.22 s
``` 

This probably relates to the [limitations when the `multiplex` plugin is used](http://avocado-framework.readthedocs.io/en/latest/RunningTestsRemotely.html?highlight=yaml#known-issues) (which is not the case for me).

**Comment by clebergnu** [comment]
@fkromer I believe this is the issue already fixed by 53afe5bd88e220dec57f5b4ae104ca1e0821181e.  Can you please verify with this commit (53afe5bd88e220dec57f5b4ae104ca1e0821181e) applied?

**Comment by fkromer** [comment]
@clebergnu I "applied" [Hacking and Using Avocado](http://avocado-framework.readthedocs.io/en/latest/ContributionGuide.html#hacking-and-using-avocado) and tried again ... works fine with [53afe5b](https://github.com/avocado-framework/avocado/commit/53afe5bd88e220dec57f5b4ae104ca1e0821181e). (Will be included in Avocado 57.0 I guess.)

**Comment by clebergnu** [comment]
Yes @fkromer, it will. Thanks for verifying the fix.
